bdjuno/README.md

```diff
@@ -23,3 +23,7 @@ docker-compose up
 BDJuno is currently being used as a submodule in which the upstream branch `chains/likecoin/mainnet` is updated and maintained by the original author Forbole.
 
 See [here](https://github.com/forbole/bdjuno/tree/chains/likecoin/mainnet) for more information
+
+### Customization
```

Comment:
instead of spreading to readme of each module, can we have a single page for customization?